### PR TITLE
Adds my blog post - is it too early to call a designer

### PR DESCRIPTION
**Pull request by rgbcolor**

### CODE REVIEW

**Comment by bcardarella** [comment]
I really like this article, but might I suggest the title change to "Is it time to call a designer?"

The current title feels like it calls a negative path to it, as it is almost trying to talk itself out of calling a designer.

**Comment by rgbcolor** [comment]
Thanks @bcardarella - I was also concerned my title was a bit too clickbait style anyhow, so your suggestion works for me. Changed!

File: source/posts/2016-04-19-is-it-time-to-call-a-designer.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+twitter: "rgbcolor"
+summary: "Since its beginnings in the 1800s, design as a profession has grown to the point that it now touches almost every aspect of our lives. But we still often assume it’s just about decoration."
+published: true
+tags: Design, UX Design, Business, Design Strategy
+---
+
+
+There is an old fable about a philosopher. A baby was born, and the parents presented the baby to the philosopher in their city at the age of just one month. They apologized for being so early, and asked for advice on how to best plan for an education so the baby would grow up as a proper citizen. To which the philosopher replied - “You’re not too early. You are ten months late.”
+
+I could not track down a proper source for this - most likely it’s an ancient Confucian fable. But I always come back to it when I plan and schedule design work. The moral of it keeps coming to mind: “you thought it was too early, but it is best to start even earlier”
```
Comment (poteto) [inline source/posts/2016-04-19-is-it-time-to-call-a-designer.md]: Missing a `.` at the end

Comment (rgbcolor) [inline source/posts/2016-04-19-is-it-time-to-call-a-designer.md]: Thanks @poteto - fixed!

**Pull request merged**